app/views/layouts/application.html.erb

```diff
@@ -3,16 +3,30 @@
   <head>
     <title>Photogram Industrial</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job adding your CDN assets to a partial! This helps keep your layout file clean :)